runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -159,6 +166,15 @@ public void hotkeyPressed()
 			hop(false);
 		}
 	};
+	private final HotkeyListener defaultKeyListener = new HotkeyListener(() -> config.defaultKey())
+	{
+		@Override
+		public void hotkeyPressed()
+		{
+			// Worlds start at 301, so add 300
+			hop(defaultWorldConfig.getWorld() + 300);
```

Comment:
Is the default world always configured to something valid?

Reply:
hop(...) checks if the world exists so I think it's ok if .getWorld() returns an invalid id